### PR TITLE
Allow installs on PHP 8

### PR DESCRIPTION
**Pull request by Jamesking56**
#2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 8.0
```
Comment (enumag) [inline .travis.yml:16]: Add 7.2, 7.3 and 7.4 as well

Comment (enumag) [reply .travis.yml:16]: Looks like tests on PHP 8 fail. Investigate what's wrong and fix it please.

Comment (Jamesking56) [reply .travis.yml:16]: Travis doesn't seem to want to pick up the build anymore

Comment (enumag) [reply .travis.yml:16]: Interesting. Sadly I have no idea why.

Comment (Jamesking56) [reply .travis.yml:16]: Nevermind its picked it up now

**Review by gharlan — APPROVED** [review]
I'm using this fork branch in my php 8.1 app. 👍

File: .travis.yml (hunk 1 above)
Comment (gharlan) [reply .travis.yml:16]: Could you add php 8.1?
And there seems to be a notice/warning since php 7.4.

Comment (Jamesking56) [reply .travis.yml:16]: Can't really now as travis.org has ceased so the CI needs rewriting (I suggest to rewrite into GitHub Actions)

**Pull request closed without merging**